spec/TEST_ASSUMPTIONS

```diff
@@ -1,9 +0,0 @@
-Before running tests we assume that manager has the following prerequisites,
-they will be added by "prerequisites" suite:
-
-1. blueprint "nodecellar1" based on nodecellar blueprint (see resources folder for the blueprint)
-2. blueprint "nodecellar_undeployed" based on nodecellar blueprint
-3. deployment "deployment_to_delete" deployed from "nodecellar1" blueprint
-5. deployment "hotkeys_deployment" deployed from "nodecellar1" blueprint
-6. deployment "installed_deployment" deployed and installed from "nodecellar1" blueprint
-4. deployment "installed_deployment2" deployed and installed from "nodecellar1" blueprint
```

Comment:
Just to have a single source of truth in `prerequisited.js` file. Because no one it's updating this TEST ASSUMPTIONS file and it shows outdated assumptions. I remember it was supposed to be deleted earlier